example_site/src/components/AccountInfoForm.js

```diff
@@ -1,16 +1,30 @@
 import { useForm } from "react-hook-form";
+import React, { useState } from "react";
 
 import { useAppContext } from "./AppProvider";
 
 export default function AccountInfoForm({
   account,
-  onSubmit,
+  updateAccount,
   formTitle,
   subtitle,
   submitButtonText,
+  canUpdate,
 }) {
   const { openSnackbar } = useAppContext();
   const { register, handleSubmit } = useForm();
+  const [buttonStatus, setButtonStatus] = useState(true);
```

Comment:
What does is mean if `buttonStatus` is true?

I think a more apt name would be something like `isButtonDisabled` since knowing if that value is `true` or `false` would give me good information about what it's controlling.